src/app.js

```diff
@@ -0,0 +1,20 @@
+import { consoleErrorhandler } from './middleware/errorHandling/errorHandling.js';
+import { initBaseRouter } from './routers';
+import express from 'express';
+import orm from './data-access/database';
+
+async function startServer() {
```

Comment:
No need to wrap all the calls here in a function startServer & call it immediately, because you use modules, which are IIFE when compiled, no need to wrap one more time